manifests/init.pp

```diff
@@ -752,6 +755,7 @@
   Optional[Stdlib::Absolutepath] $server_versioned_code_id = undef,
   Optional[Stdlib::Absolutepath] $server_versioned_code_content = undef,
   Array[String[1]] $server_jolokia_metrics_whitelist = [],
+  Pattern[/^[0-9]{3,4}$/] $puppetconf_mode = $puppet::params::puppetconf_mode,
```

Comment:
Would `Stdlib::Filemode` be better?
```suggestion
  Stdlib::Filemode $puppetconf_mode = $puppet::params::puppetconf_mode,
```

Reply:
Thanks for pointing this out, I was not aware this existed!